cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -82,3 +82,259 @@ def test_provider_summary_topology():
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select at least 2 hosts by checking the box in upper left of host icons.
+            3. Click "Compare Selected Items" under the "Configuration" dropdown.
+            4. Click on "Host Properties(X)" in the Compare Host view.
+            5. Click on "Host Properties(X)" again, in the Compare Host view.
+            6. Click to expand the "Properties" comparison section, select "Hardware", and click
+             "Apply".
+            7. Click on "Hardware(X)" in the Compare Host view.
+            8. Click on "Hardware(X)" again, in the Compare Host view.
+            9. Click to expand the "Properties" comparison section, select "Network Adapters", and
+            click "Apply".
+            10. Click on "Network Adapters(X)" in the Compare Host view.
+            11. Click on the "#X" items that apply to the network adapters.
+            12. Click on the "#X" items again that apply to the network adapters.
+            13. Click on "Network Adapters(X)" again, in the Compare Host view.
+        expectedResults:
+            1. Hosts view is displayed with hosts filtered via the Default filter (denoted by
+            "(Default)" next to the filter name in the dropdown).
+            2. The selected hosts should be displayed with a blue border and checked checkbox.
+            3. The "Compare Host / Node" view should be displayed.
+            - icons are displayed for all and only selected hosts with hostname displayed
+            - one of the host icons has the host denoted as "(base)" in the hostname
+            - host properties row is displayed (default)
+            - "% Matched" text or graphs are displayed
+            - when 3 or more hosts are displayed, remove icons exist for all non-base hosts.
+            4. The row should be expanded to display all of the properties compared. Items that do
+            not match the base
+            host should be in blue. There should be X properties displayed. Properties for non-base
+            hosts should be in
+            purple/dark blue.
+            5. The Host properties should collapse to one row again.
+            6. A hardware row should be added to the view for all hosts with % matching graphs
+            displayed for non-base
+            hosts.
+            7. The row should be expanded and displayed with same requirements as in step 4.
+            8. The hardware metrics should collapse to one row again.
+            9. A network adapters row should be added to the view for all hosts with % matching
+            graphs displayed for
+            non-base hosts.
+            10. The row should be expanded and displayed with same requirements as in step 4.
+            11. The "#X" individual network adapter rows should be expanded and displayed with
+            same requirements as in
+            step 4.
```

Comment:
Kindly fix the indentation in L168-169 and similar lines above. Some of that text can fit on same line, instead of adding it on a newline.